### PR TITLE
remove shell restrictions

### PR DESCRIPTION
**Pull request by kcin2001**
unlike on virgo (last i checked) borgs and ai have to be whitelisted so why not trust them to use their power responsibly instead of preventing them from using arbitrary modules for their shell and instead let them choose what would be most usefull (or aestetically pleasing)

### CODE REVIEW

**Comment by kcin2001** [comment]
discussion encouraged

**Comment by kcin2001** [comment]
oh yeah ill do that

**Comment by TheFurryFeline** [comment]
Now that I've opened up DM to test this myself as a peer review, compiling throws up a few errors.  `	updatename()
	notify_ai(ROBOT_NOTIFICATION_NEW_MODULE, module.name)

/mob/living/silicon/robot/proc/updatename(var/prefix as text)` 
code\modules\mob\living\silicon\robot\robot.dm:285:error: bad argument definition
code\modules\mob\living\silicon\robot\robot.dm:285:error: bad argument definition
code\modules\mob\living\silicon\robot\robot.dm:288:error: bad argument definition

**Comment by TheFurryFeline** [comment]
Also, took the opportunity to look at Virgo's PR. They added a global_vr.dm code snippet 
shown in https://github.com/VOREStation/VOREStation/pull/5696/files 

In there, the old version of robot.dm was 
```
	modules.Add(robot_module_types)
	if(crisis || security_level == SEC_LEVEL_RED || crisis_override) // VOREStation Edit
		to_chat(src, "<font color='red'>Crisis mode active. Combat module available.</font>")			
		modules+="Combat"		else
		modules+="ERT" //VOREStation Edit	
```

**Comment by kcin2001** [comment]
i cannot verify it in a test server because dream daemon is being shy right now

**Comment by TheFurryFeline** [comment]
Sorry, chief: 

> vorestation.dmb - 83 errors, 17 warnings (4/9/20 12:10 pm)

**Comment by kcin2001** [comment]
what the, ugh lemme see if doing it the proper way creates anything else in the commit

**Comment by Razgriz1032** [comment]
Nadyr and I talked, we'd like to see restrictions placed on science models so that the AI can't just keep printing themselves new shells without at least some help.

**Comment by TheFurryFeline** [comment]
```
var/list/shell_module_types = list(
	"Standard", "Engineering", "Surgeon",  "Crisis",
	"Miner", "Janitor", "Service", "Clerical",
	"Medihound", "Janihound",
	"Servicehound" //YW changes
``` in https://github.com/CHOMPStation2/CHOMPStation2/blob/ab9e1f1c4c430c0064be58e5ba0f00567fd837ec/code/global_vr.dm#L11 is what you want to cha-waaaaait a second, it looks like science was already removed? Hold on, lemme test this, then.

**Comment by kcin2001** [comment]
That's easy to do id change the list to be just the sciborg and the other one and change the code to a blacklist

**Comment by kcin2001** [comment]
a blacklist is easier to maintain than a whitelist imo

**Comment by TheFurryFeline** [comment]
Tested it, that code does not seem to work properly to remove research module and sci-borg for shell selection list.

**Comment by kcin2001** [comment]
requested changes have been added

File: code/global_vr.dm [hunk 1]
```diff
@@ -7,14 +7,14 @@
 	robot_module_types += "Servicehound" //YW changes
 	robot_module_types += "BoozeHound" //YW changes
 	return 1
-
+/*
```
Comment (TheFurryFeline) [inline code/global_vr.dm:10]: //ch removal comment

Comment (TheFurryFeline) [reply code/global_vr.dm:10]: Just realised mobile doesn't quite work well for this. Comment not showing what's below that line.  D'oh, am sleepy

**Comment by kcin2001** [comment]
and if any other modules are deemed unsuitable for ai shells its just a matter of adding its named to the blacklist

**Comment by Razgriz1032** [comment]
Looks good thank you!

**Pull request merged**